docs/Testing.md

```diff
@@ -62,7 +62,9 @@ Caffe2](https://caffe2.ai/docs/operators-catalogue.html#recurrentnetwork)), the
 input model must be unrolled to some maximum input and output length. These can
 be specified on the command line via `-min_output_len` and
 `-max_output_len`. Additionally, the beam search size can be specified via
-`-beam_size`.
+`-beam_size`. The default options for the `text-translator` match those for the
+model currently downloaded via `utils/download_caffe2_models.sh`
```

Comment:
please add `en2gr`: for the en2gr model currently downloaded ...

Reply:
Done.